highfive/newpr.py

```diff
@@ -113,6 +113,7 @@ def set_assignee(self, assignee, owner, repo, issue, user, author, to_mention):
             if e.code == 201:
                 pass
             else:
+                print(f"failed to assign {assignee} to {owner}/{repo}#{issue}")
```

Comment:
Hm we might want to not log on @ghost because that's needless noise

Reply:
Well @ghost also prints the 10/15 log entries of the stack trace, an extra entry won't make things less hard to understand :(

Reply:
Ah. Well. Okay.